doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -83,14 +110,12 @@ Proposals (WK): Change title of this section to 2.3 Architectural Requirements;
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.int.gen.01` | Interface | The Architecture shall/must xxx. |
-| `req.int.gen.02` | Interface | The Architecture shall/must xxx. |
-| `req.int.gen.03` | Interface | The Architecture shall/must xxx. |
-| `req.int.api.01` | API | The Architecture **must** provide Control API endpoints, GUI & internal services for the cloud |
-| `req.int.api.02` | API | The Architecture **shall** allow external access to APIs exposed by the VIM and NFVI. |
-| `req.int.api.03` | API | The Architecture shall/must xxx. |
-| `req.int.acc.01` | Acceleration | The Architecture **shall** provide an open and standard acceleration interface to VNFs. |
-| `req.int.acc.02` | Acceleration | The Architecture **shall not** rely on SR-IOV PCI-Pass through for acceleration interface exposed to VNFs. |
+| `req.int.api.01` | API | The Architecture **must** provide Control API endpoints to cloud platform core services |
+| `req.int.api.02` | API | The Architecture **must** provide GUI access to tenant facing cloud platform core services |
+| `req.int.api.03` | API | The Architecture **should** allow enabling external access to Control API endpoints and GUI. |
```

Comment:
What is meant with "external access" in the context of NFVI?

Reply:
@hdamker this is more of a VIM requirement than VFVI. i.e. access to API endpoints externally (from the internet) when needed.